includes/class-give-scripts.php

```diff
@@ -193,8 +190,10 @@ private function plugin_equeue_scripts() {
 			'submit_and_deactivate'           => __( 'Submit and Deactivate', 'give' ),
 			'skip_and_deactivate'             => __( 'Skip & Deactivate', 'give' ),
 			'please_fill_field'               => __( 'Error: Please fill the field.', 'give' ),
+			'ajaxurl'                         => give_get_ajax_url(),
```

Comment:
@raftaar1191 remove this, WordPress provides `ajaxurl` in admin pages

Reply:
fix @ravinderk